public/lang/de.json

```diff
@@ -1 +1,158 @@
+        "accept": "Akzeptieren",
+        "decline": "Ablehnen",
+        "title-image": "Bild",
+        "title-file": "Datei",
+        "title-image-plural": "Bilder",
+        "title-file-plural": "Dateien",
+        "scan-qr-code": "oder scanne den QR-Code.",
+        "would-like-to-share": "möchte Folgendes teilen",
+        "send": "Senden",
+        "copy": "Kopieren",
+        "receive-text-title": "Textnachricht erhalten",
+        "file-other-description-image-plural": "und {{count}} andere Bilder",
+        "file-other-description-file-plural": "und {{count}} andere Dateien",
+        "activate-paste-mode-activate-paste-mode-shared-text": "Freigegebener Text",
+        "auto-accept-instructions-1": "Aktiviere",
+        "auto-accept": "auto-accept",
+        "auto-accept-instructions-2": "um automatisch alle Dateien von diesem Gerät zu akzeptieren.",
+        "has-sent": "hat Folgendes gesendet:",
+        "send-message-title": "Textnachricht senden",
+        "send-message-to": "Sende eine Textnachricht an",
+        "base64-tap-to-paste": "Tippe hier um {{type}} einzufügen",
+        "base64-paste-to-send": "Füge {{type}} hier zum Versenden ein",
+        "base64-text": "Text",
+        "base64-files": "Dateien",
+        "base64-processing": "Bearbeitung läuft…",
+        "file-other-description-image": "und 1 anderes Bild",
+        "file-other-description-file": "und 1 andere Datei",
+        "receive-title": "{{descriptor}} erhalten",
+        "download-again": "Erneut herunterladen",
+        "system-language": "Systemsprache",
+        "language-selector-title": "Sprache auswählen",
+        "hr-or": "ODER",
+        "input-room-id-on-another-device": "Gebe diese Raum ID auf einem anderen Gerät ein",
+        "unpair": "Entkoppeln"
+    },
+    "about": {
+        "tweet_title": "Über PairDrop twittern",
+        "faq_title": "Häufig gestellte Fragen",
+        "close-about_aria-label": "Schließe Über PairDrop",
+        "github_title": "PairDrop auf GitHub",
+        "buy-me-a-coffee_title": "Kauf mir einen Kaffee!",
+        "claim": "Der einfachste Weg Dateien zwischen Geräten zu teilen"
+    },
+    "footer": {
+        "known-as": "Du wirst angezeigt als:",
+        "display-name_title": "Setze einen permanenten Gerätenamen",
+        "discovery-everyone": "Du bist erkennbar von jedem",
+        "on-this-network": "in diesem Netzwerk",
+        "and-by": "und von",
+        "paired-devices": "von gekoppelten Geräten",
+        "traffic": "Datenverkehr wird",
+        "display-name_placeholder": "Lade…",
+        "routed": "durch den Server geleitet",
+        "webrtc": "wenn WebRTC nicht verfügbar ist.",
+        "display-name_data-placeholder": "Lade…",
+        "public-room-devices_title": "Du kannst von Geräten in diesem öffentlichen Raum unabhängig von deinem Netzwerk gefunden werden.",
+        "paired-devices_title": "Du kannst immer von gepaarten Geräten egal in welchem Netzwerk gefunden werden.",
```

Comment:
```suggestion
        "paired-devices_title": "Du kannst immer von gepaarten Geräten, egal in welchem Netzwerk gefunden werden.",
```

Reply:
I will switch it around:
`Du kannst immer von gepaarten Geräten gefunden werden, egal in welchem Netzwerk.`